code/game/objects/effects/portals.dm

```diff
@@ -147,7 +147,7 @@
 	var/turf/real_target = get_link_target_turf()
 	if(!istype(real_target))
 		return FALSE
-	if(!ismecha(M) && M.anchored && !allow_anchored)
+	if(!ismecha(M) && !istype(M, /obj/item/projectile) && M.anchored && !allow_anchored)

```

Comment:
Why are you making it so we can't shoot lasers through portals to the other side

Reply:
goof you can't read code this is making it so you can

Reply:
Fuck

Reply:
tbf that's what I thought too when I closed the original. The title isn't that descriptive